savant/gstreamer/metadata.py

```diff
@@ -48,6 +48,61 @@ def clear(self) -> None:
         )
 
 
+class InitialSizeTransformation(NamedTuple):
+    width: int
+    height: int
+
+    def to_dict(self):
+        return {**self._asdict(), 'type': 'initial_size'}
+
+
+class ResultingSizeTransformation(NamedTuple):
+    width: int
+    height: int
+
+    def to_dict(self):
+        return {**self._asdict(), 'type': 'resulting_size'}
+
+
+class ScaleTransformation(NamedTuple):
+    width: int
+    height: int
+
+    def to_dict(self):
+        return {**self._asdict(), 'type': 'scale'}
+
+
+class PaddingTransformation(NamedTuple):
+    left: int
+    top: int
+    right: int
+    bottom: int
+
+    def to_dict(self):
+        return {**self._asdict(), 'type': 'padding'}
+
+
+Transformation = Union[
+    InitialSizeTransformation,
+    ResultingSizeTransformation,
+    ScaleTransformation,
+    PaddingTransformation,
+]
```

Comment:
Are these types useful?

As I see it, the transformation tuples are built from dicts and are consumed as dicts. Can they just be dicts?